init-tools.cmd

```diff
@@ -46,6 +46,15 @@ if exist "%DotNetBuildToolsDir%" (
 echo Running %0 > "%INIT_TOOLS_LOG%"
 
 set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.txt"
+
+:DotNetVersionLoop
```

Comment:
Do we really need to use gotos? Can't we just say:
```
set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.txt"
if /i [%1] == [-Debug-Intrinsics] ( set DOTNET_VERSION=3.0.100-alpha1-009622)
if /i [%1] == [-Release-Intrinsics] ( set DOTNET_VERSION=3.0.100-alpha1-009622)
```

Reply:
we can remove sum of them. we need to loop through all the cmd line arguments to check if the user has supplied us with the deseried argument
```
:Arg_Loop
if [%1] == [] goto :ArchSet
if /i [%1] == [x86]         ( set ARCH=x86)
if /i [%1] == [-Debug-Intrinsics] ( set DOTNET_VERSION=3.0.100-alpha1-009622)
if /i [%1] == [-Release-Intrinsics] ( set DOTNET_VERSION=3.0.100-alpha1-009622)
shift
goto :Arg_Loop
```

Reply:
Ah makes sense. Disregard then.